wp-includes/sqlite/class-wp-sqlite-translator.php

```diff
@@ -888,6 +888,8 @@ private function execute_create_table() {
 			')'
 		);
 
+		$if_not_exists = false === strpos( $create_query, 'IF NOT EXISTS' ) ? '' : 'IF NOT EXISTS';
```

Comment:
@akirk, something just occurred to me that probably affects other parts of this plugin as well:
Aren't these keywords case insensitive, and if so, we should probably make the check case insensitive.

Reply:
Well, there is also potential whitespace deviations. Either [we normalize the output here](https://github.com/WordPress/sqlite-database-integration/blob/develop/wp-includes/sqlite/class-wp-sqlite-query-rewriter.php#L84C18-L84C35), or we use a regex with `\s+` instead of spaces.

Reply:
We could also use `stripos` instead of `strpos` to make this case-insensitive... I'd rather we limit the use of regex if not absolutely necessary.

Reply:
But what if someone writes ` if NOT   exists`? One other option is to change `get_updated_query()` so that it will return normalized SQL, like unify whitespace, uppercase lexer tokens, then we can just use `strpos`.

Reply:
Fair point 👍